test/controllers/storages_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class StoragesControllerTest < ActionController::TestCase
+  setup do
+    @storage = storages(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:storages)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create storage" do
+    assert_difference('Storage.count') do
+      post :create, storage: { title: @storage.title, url: @storage.url }
+    end
+
+    assert_redirected_to storage_path(assigns(:storage))
+  end
+
+  test "should show storage" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.